src/components/SecondaryPanes/Frames/Group.css

```diff
@@ -35,3 +35,16 @@
 .frames ul .frames-group.expanded .badge {
   color: var(--theme-highlight-blue);
 }
+
+.arrow {
+  display: inline-block;
+  width: 12px;
+  margin-inline-end: 8px;
+  transform: rotate(0);
+  transform-origin: center center;
+  transition: transform 0.125s ease;
+}
```

Comment:
did you try using the other Arrow components? I feel like this is probably defined elsewhere

Reply:
I feel like we could re-use what we already have in various places and harmonize everything: 

```diff
diff --git a/src/components/PrimaryPanes/Sources.css b/src/components/PrimaryPanes/Sources.css
index e88b27f64..ba288d930 100644
--- a/src/components/PrimaryPanes/Sources.css
+++ b/src/components/PrimaryPanes/Sources.css
@@ -103,10 +103,6 @@
   color: inherit;
 }
 
-.sources-list .tree .img.arrow.expanded {
-  transform: rotate(0deg);
-}
-
 .theme-dark .source-list .tree .node.focused {
   background-color: var(--theme-tab-toolbar-background);
 }
diff --git a/src/components/SecondaryPanes/Frames/Group.css b/src/components/SecondaryPanes/Frames/Group.css
index 0da19980b..424203523 100644
--- a/src/components/SecondaryPanes/Frames/Group.css
+++ b/src/components/SecondaryPanes/Frames/Group.css
@@ -39,16 +39,3 @@
 .group-description-name {
   padding-left: 5px;
 }
-
-.arrow {
-  display: inline-block;
-  width: 12px;
-  margin-inline-end: 8px;
-  transform: rotate(0);
-  transform-origin: center center;
-  transition: transform 0.125s ease;
-}
-
-html[dir="rtl"] .arrow.expanded {
-  transform: rotate(90deg);
-}
diff --git a/src/components/SecondaryPanes/Frames/Group.js b/src/components/SecondaryPanes/Frames/Group.js
index 5f826055c..667b3013f 100644
--- a/src/components/SecondaryPanes/Frames/Group.js
+++ b/src/components/SecondaryPanes/Frames/Group.js
@@ -9,6 +9,7 @@ import classNames from "classnames";
 import Svg from "../../shared/Svg";
 import { getLibraryFromUrl } from "../../../utils/pause/frames";
 import FrameMenu from "./FrameMenu";
+import AccessibleImage from "../../shared/AccessibleImage";
 
 import "./Group.css";
 
@@ -17,16 +18,17 @@ import FrameComponent from "./Frame";
 import type { LocalFrame } from "./types";
 import Badge from "../../shared/Badge";
 
-type FrameLocationProps = { frame: LocalFrame, expandedState: boolean };
-function FrameLocation({ frame, expandedState }: FrameLocationProps) {
+type FrameLocationProps = { frame: LocalFrame, expanded: boolean };
+function FrameLocation({ frame, expanded }: FrameLocationProps) {
   const library = frame.library || getLibraryFromUrl(frame);
   if (!library) {
     return null;
   }
-  const classArrowExpanded = expandedState ? "arrow expanded" : "arrow";
+
+  const arrowClassName = classNames("arrow", { expanded });
   return (
     <span className="group-description">
-      <Svg name="arrow" className={classArrowExpanded} />
+      <AccessibleImage className={arrowClassName} />
       <Svg name={library.toLowerCase()} className="annotation-logo" />
       <span className="group-description-name">{library}</span>
     </span>
@@ -142,7 +144,7 @@ export default class Group extends Component<Props, State> {
         tabIndex={0}
         title={title}
       >
-        <FrameLocation frame={frame} expandedState={expanded} />
+        <FrameLocation frame={frame} expanded={expanded} />
         <Badge>{this.props.group.length}</Badge>
       </li>
     );
diff --git a/src/components/shared/AccessibleImage.css b/src/components/shared/AccessibleImage.css
index 4810bdbcb..6617dcd33 100644
--- a/src/components/shared/AccessibleImage.css
+++ b/src/components/shared/AccessibleImage.css
@@ -5,3 +5,7 @@
   /* makes span appear like an image */
   display: inline-block;
 }
+
+.img.arrow.arrow.expanded {
+  transform: rotate(0deg);
+}
```

This way, everything is still working, and we don't repeat all the CSS properties.